bin/run_bnb_trades.sh

```diff
@@ -0,0 +1,4 @@
+ #! /bin/sh
+
+BLOCKCHAIN=bnb NODE_URL=https://dex.binance.org/api/v1/trades/ BNB_MODE=trades \
+docker-compose -f docker/docker-compose.yaml up --build && docker-compose -f docker/docker-compose.yaml rm -f
```

Comment:
what do you think of `set -e`. So you can avoid `&&` here

Reply:
Sounds good. Done.